.github/workflows/prod-release.yaml

```diff
@@ -0,0 +1,62 @@
+name: seqr prod release
```

Comment:
Do we need this, as we're not using K8s?

Reply:
Reasonable call, actioned here: https://github.com/populationgenomics/seqr/pull/174